### PR TITLE
Remove all `{{SpecName}}` macros from l10n-zh

### PR DESCRIPTION
**Pull request by yin1999**
exclude `conflicting` and `orphaned` folders

poke @irvin for a review on zh-tw files

### CODE REVIEW

File: files/zh-cn/web/api/document/visibilitychange_event/index.html [hunk 1]
```diff
@@ -14,8 +14,6 @@
 <h2 id="概述">概述</h2>
 
 <dl>
- <dt>Specification</dt>
- <dd>{{SpecName("Page Visibility API")}}</dd>
```
Comment (yin1999) [inline files/zh-cn/web/api/document/visibilitychange_event/index.html:18]: there as specifiction table below

File: files/zh-cn/web/css/color_value/index.html [hunk 2]
```diff
@@ -75,908 +75,12 @@ <h3 id="颜色关键字">颜色关键字</h3>
 </ul>
 </div>
 
-<table id="colors_table">
```
Comment (yin1999) [inline files/zh-cn/web/css/color_value/index.html:78]: color table is removed from en-US

File: files/zh-cn/web/javascript/guide/regular_expressions/assertions/index.html [hunk 3]
```diff
@@ -230,23 +230,6 @@ <h3 id="向后断言">向后断言</h3>
 let ripe_oranges = oranges.filter( fruit =&gt; fruit.match(/(?&lt;=ripe )orange/));
 console.log(ripe_oranges); // [ 'ripe orange A ', 'ripe orange C' ]</pre>
 
-<h2 id="规范">规范</h2>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/guide/regular_expressions/assertions/index.html:233]: removed from en-US

File: files/zh-tw/glossary/css_selector/index.md [hunk 4]
```diff
@@ -52,7 +52,3 @@ p {
 
 - [虛擬元素](/zh-TW/docs/Web/CSS/Pseudo-elements)
 - [虛擬類別](/zh-TW/docs/Web/CSS/Pseudo-classes)
-
-### 技術指南
-
-{{SpecName("CSS3 Selectors")}}
```
Comment (yin1999) [inline files/zh-tw/glossary/css_selector/index.md:58]: removed from en-US

File: files/zh-tw/web/api/performance/timing/index.html [hunk 5]
```diff
@@ -25,30 +25,11 @@ <h2 id="語法">語法</h2>
...
-   <th scope="col">備註</th>
-  </tr>
-  <tr>
-   <td>{{SpecName('Navigation Timing', '#sec-window.performance-attribute', 'Performance.timing')}}</td>
-   <td>{{Spec2('Navigation Timing')}}</td>
-   <td>初步定義</td>
-  </tr>
- </tbody>
-</table>
+<p>因為 <a>Navigation Timing 規範</a>已被棄用，此特性不再有望成為標準。請使用 {{domxref("PerformanceNavigationTiming")}} 介面代替。</p>
```
Comment (yin1999) [inline files/zh-tw/web/api/performance/timing/index.html:28]: please check the translation

File: files/zh-tw/web/api/proximity_events/index.html [hunk 6]
```diff
@@ -27,24 +27,9 @@ <h2 id="範例">範例</h2>
...
-  <tr>
-   <td>{{ SpecName('Proximity Events', '', 'Proximity Events') }}</td>
-   <td>{{ Spec2('Proximity Events') }}</td>
-   <td>Initial specification</td>
-  </tr>
- </tbody>
-</table>
+<h2 id="Specifications">規範</h2>
 
+<p>此功能不再有望成為標準。它的歷史規範仍然<a href="https://w3c.github.io/proximity/">可用</a>。</p>
```
Comment (yin1999) [inline files/zh-tw/web/api/proximity_events/index.html:32]: please check

**Comment by irvin** [comment]
@yin1999 thanks!

**Pull request merged**